server/main.py

```diff
@@ -1,91 +1,297 @@
 from fastapi import FastAPI
 from fastapi import HTTPException
-
+import uvicorn
+from typing import Optional
+from pydantic import BaseModel
 import storage
 import schemas
+import json
 
 
 app = FastAPI()
 
 
-@app.get("/list-games/")
-async def list_games():
-    return {
-        "games": storage.get_games(),
-    }
+# @app.post("/register-as-a-player/")
+# async def register_as_a_player(player_id_1: str, player_id_2: str):
+#     return storage.register_player(player_id_1, player_id_2)
+###########################################################
+###########################################################
+_GAMES_STORAGE = {
+    1: {
+        "id": 1,
+        "player_id_1": "A",
+        "player_id_2": "B",
+        "status": "new",
+        "current_move": "x",
+        "player_score_1": 10,
+        "player_score_2": 5,
+        "field": [
+            ["", "", ""],
+            ["", "", ""],
+            ["", "", ""],
+        ],
+    },
 
+    2: {
+        "id": 2,
+        "player_id_1": "A",
+        "player_id_2": "B",
+        "status": "new",
+        "current_move": "x",
+        "player_score_1": 10,
+        "player_score_2": 5,
+        "field": [
+            ["", "", ""],
+            ["", "", ""],
+            ["", "", ""],
+        ],
+    },
+}
 
-@app.post("/create-game/")
-async def create_game():
-    return storage.create_game()
 
+class new_games(BaseModel):
 
-def check_winner(game: dict) -> str | None:
-    """
+    id: int
+    player_id_1: (str) = ""
+    player_id_2: (str) = ""
+    status: str = "new"
+    current_move: str = "x"
+    player_score_1: float = 0
+    player_score_2: float = 0
+    field = [
+        ["", "", ""],
+        ["", "", ""],
+        ["", "", ""],
+    ],
+
+
+@app.post('/create-item/{game_id}')
+def add_new_item(game_id: int, new_games: new_games):
+    if game_id in _GAMES_STORAGE:
+        raise HTTPException(
+            status_code=404, detail=f"the id {game_id} already exists")
+
+    _GAMES_STORAGE[game_id] = new_games
+    return _GAMES_STORAGE[game_id]
+
+
+@app.get("/get-game/{game_id}")  # item_id is called path parameter
+# path gives a description of the path parameter
+# gt  lt ge and le :make restrictions on the path parameter to be in a specific interval gives 422 error
+def get_game(game_id: int):
+    if game_id in _GAMES_STORAGE:
+        return _GAMES_STORAGE[game_id]
+    raise HTTPException(
+        status_code=404, detail=f"no game with  id {game_id} ")
+
+
+class update_item(BaseModel):
```

Comment:
Class should be named `UpdateItem` according to PEP8. Also, the name is too generic, it's not clear which item it supposed to update. Moreover, you would need to update more than one item type, how would you call a class for this, `UpdateItem2`? Perhaps it's not the best idea.